scripts/docker-as-drupal

```diff
@@ -178,6 +179,12 @@ function setup() {
     cp $SERVICES_FILE ./web/sites/default/services.yml
   fi
 
+  # Setup drush.yml file
+  if [ ! -z "$DRUSH_FILE" ] && [ -f "$DRUSH_FILE" ] ; then
+    printf "\e[1;35m* Copy drush.yml file.\e[0m\n"
+    mkdir -p "./drush" && cp $DRUSH_FILE ./drush/drush.yml
```

Comment:
Why do you need this?

Reply:
mkdir ?

Reply:
because the root `.drush` does not exists in our repository